modules/migration-creating-ca-bundle.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// migration/migrating_3_4/migrating-applications-with-cam.adoc
+// migration/migrating_4_1_4/migrating-applications-with-cam.adoc
+// migration/migrating_4_2_4/migrating-applications-with-cam.adoc
+[id='creating-ca-bundle_{context}']
+= Creating a CA certificate bundle file
+
+If you use a self-signed certificate to secure a cluster or a replication repository, certificate verification may fail with the following error message, `certificate signed by unknown authority`.
```

Comment:
The error message should follow a colon with the correct capitalization:  
"....following error message: `Certificate signed by unknown authority`."